examples/models/modules/layers_3d.py

```diff
@@ -0,0 +1,84 @@
+from torch import nn
+
+from torchsparse import nn as spnn
+
+__all__ = ['SparseConvBlock', 'SparseDeConvBlock', 'SparseResBlock']
+
+
+class SparseConvBlock(nn.Module):
+
+    def __init__(self,
+                 in_channels: int,
+                 out_channels: int,
+                 kernel_size,
+                 stride=1,
+                 dilation: int = 1) -> None:
+        super().__init__()
+        self.net = nn.Sequential(
+            spnn.Conv3d(in_channels,
+                        out_channels,
+                        kernel_size,
+                        stride=stride,
+                        dilation=dilation), spnn.BatchNorm(out_channels),
+            spnn.ReLU(True))
+
+    def forward(self, x):
+        return self.net(x)
+
+
+class SparseDeConvBlock(nn.Module):
```

Comment:
Same here.

Reply:
Also, let's rename this block as `SparseConvTransposeBlock` following PyTorch naming convention.